docs/src/distributed/namedservices.md

```diff
@@ -0,0 +1,105 @@
+# Named Services
+
+In the last section we discussed how to build a leader election mechanism with a bunch of networked Kompact systems. But we couldn't actually run it in deployment, because we couldn't really figure out how to collect a list of actor paths for all the processes and then distributed that list to every process. This happens because we can only know the actor path of an actor *after* we have created it. We could have manually distributed the actor paths, by writing the assigned path to a file, then collecting it externally, and finally parsing paths from said collected file and passing them to each elector component. But that wouldn't be a very nice system now, would it? 
+
+What we are missing here is a way to predict an `ActorPath` for a particular actor on a particular system. If we can know even a single path on a single host in the distributde actor system, we can have everyone send a message there, which will give that special component the unique paths for everyone that sends there, which it can in turn distribute back to everyone who has "checked in" in this manner. This process is often referred to as "bootrapping". In this section we are going to use named actor paths, which we can predict given some information about the system, to build a bootstrapping "service" for our leader election group.
```

Comment:
typos in this paragraph: distributde -> distributed, bootrapping -> bootstrapping.